src/main/webapp/css/snyk_report.css

```diff
@@ -192,6 +192,11 @@ a[href^="https://"]:after {
   vertical-align: text-top;
 }
 
+.card .label--critical {
```

Comment:
Not sure we even need this file @pavel-snyk thinks we may actually download snyk-to-html - so perhaps this can be deleted??? TBC